packages/devops/scripts/ci/copyInEnvironmentVariables.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+# So long as setupEnvironmentVariables has run, the common volume "environment_variables"
+# will have all .env files in the same directory structure as they're expected within the
+# tupaia repo, so we just rsync the whole volume to copy the .env files in
+
+rsync -a /environment_variables/ /tupaia/
```

Comment:
I never knew about `rsync`! Seems really cool 🎉

Reply:
It is! Made this quite straightforward